src/components/no-results-message/noResultsMessage.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import { Icon } from '@folio/stripes/components';
+
+import css from './noResultsMessage.css';
+
+const propTypes = {
+  children: PropTypes.node.isRequired,
+};
+
+const NoResultsMessage = ({ children, ...rest }) => {
+  return (
+    <div className={css.noResultsMessage} {...rest}>
+      <div className={css.noResultsMessageLabelWrap}>
```

Comment:
@BogdanDenis, @MaksymDryha Should we replace this tag with `<p>` like in `SearchAndSort` component?

Reply:
@Vladyslav-Velytskyi , makes sense